crucible/src/Lang/Crucible/CFG/SSAConversion.hs

```diff
@@ -826,8 +883,15 @@ resolveStmts nm bi sz reg_map bindings appMap (Posd p s0:rest) t = do
                            (resolveAtom reg_map m))
                  (resolveStmts nm bi sz reg_map bindings appMap rest t)
 
+    Breakpoint{} -> error $
```

Comment:
Let's add a comment explaining why we don't expect to see any `Breakpoint` statements here.